recipes/mtspec/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.3.0" %}
+{% set sha256 = "307dc34acefdfae06f055d99d0260cb3529a1fd46d7696be2ab533752aff01a1" %}
+
+package:
+  name: mtspec
+  version: {{ version }}
+
+
+source:
+  fn: mtspec-{{ version }}.zip
+  url: https://files.pythonhosted.org/packages/e8/23/a6b16605dbe1e47a2a6128957ffd4c6d3aa06bd2971de4de3825e1f6f65d/mtspec-0.3.0.zip
```

Comment:
Can you change that to: `https://pypi.io/packages/source/m/mtspec/mtspec-{{ version }}.zip`

(The new PyPI URLs are horrible to read :smile: )